README.md

```diff
@@ -23,4 +23,6 @@ read me
 [] Item 11
 
 ## Title 2
-* Olá | Hello | Konnichiwa | 
\ No newline at end of file
+* Olá | Hello | Konnichiwa | 
+
+``console.log(1+3)``
```

Comment:
Hi there, nice job! Could you be add one more example of code in the READ.ME, please?

Reply:
Sure. I'll add

Reply:
Done